src/shared/utils/isTextInputElement.js

```diff
@@ -32,12 +33,18 @@ var supportedInputTypes = {
   'week': true,
 };
 
-function isTextInputElement(elem) {
+function isTextInputElement(elem: ?HTMLElement): bool {
   var nodeName = elem && elem.nodeName && elem.nodeName.toLowerCase();
-  return nodeName && (
-    (nodeName === 'input' && supportedInputTypes[elem.type]) ||
-    nodeName === 'textarea'
-  );
+
+  if (nodeName === 'input') {
```

Comment:
If if is, I would also do the same for textarea instead of doing nodeName.toLowercase()